google-cloud-compute-v1/lib/google/cloud/compute/v1/accelerator_types/rest/client.rb

```diff
@@ -0,0 +1,452 @@
+# frozen_string_literal: true
+
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Auto-generated by gapic-generator-ruby. DO NOT EDIT!
+
+require "google/cloud/errors"
+require "google/cloud/compute/v1/compute_small_pb"
+
+module Google
+  module Cloud
+    module Compute
+      module V1
+        module AcceleratorTypes
+          module Rest
+            ##
+            # REST client for the AcceleratorTypes service.
+            #
+            # Services
+            #
+            # The AcceleratorTypes API.
+            #
+            class Client
+              include GrpcTranscoding
```

Comment:
Probably fine for Alpha, but just a note that this risks collisions between generated client methods and generated transcoding methods. (e.g. if a particular service has a method named `transcode_something`)

Reply:
noted